biosdrain.c

```diff
@@ -170,6 +190,7 @@ int main(void)
 	graphic_biosdrain_fadein();
 
 	menu_status("biosdrain - revision %s\n", GIT_VERSION);
+	menu_status("version with namco system 246/256 (COH-H models) support\n");
```

Comment:
Might want to hide this behind the `SUPPORT_SYSTEM_2x6` define as well

Reply:
Forgot about this too.
Will do it when I finish the pr